entrypoint/entrypoint.go

```diff
@@ -34,11 +36,17 @@ func RunApp(app *cli.App) {
 }
 
 // If there is an error, display it in the console and exit with a non-zero exit code. Otherwise, exit 0.
+// Note that if the GRUNTWORK_DEBUG environment variable is set to 1, this will print out the stack trace.
 func checkForErrorsAndExit(err error) {
 	exitCode := defaultSuccessExitCode
+	isDebugMode := os.Getenv(debugEnvironmentVarName) == "1"
```

Comment:
Maybe if the var is not an empty string, we enable debug mode? I'll be tempted to set it to `true` or similar...